src/pkg/packager/common.go

```diff
@@ -188,6 +189,14 @@ func (p *Packager) attemptClusterChecks(ctx context.Context) error {
 		}
 	}
 
+	state, err := p.cluster.LoadZarfState(ctx)
+	// don't return the err here as state may not yet be setup
+	if err == nil {
+		if err := pki.CheckForExpiredCert(ctx, state.AgentTLS); err != nil {
+			return err
+		}
+	}
```

Comment:
Nit, but I'm a big fan of early return:
```golang
state, err := p.cluster.LoadZarfState(ctx)
if err != nil {
    // don't return the err here as state may not yet be setup
    return nil
}
if err := pki.CheckForExpiredCert(ctx, state.AgentTLS); err != nil {
    return err
}
```
The other benefit is that you're clearly stating why and where you're ingoring the error, with your version I was a bit confused and thought you're ignoring the error.